### PR TITLE
Fixbug/tiemlv/fixbug 268 show last message when typing

### PR DESCRIPTION
**Pull request by TiemLuong**
### Pull Request Checklist

* [ ] Pull request is based on the develop branch
* [ ] Pull request updates [CHANGES.rst](https://github.com/vector-im/riot-ios/blob/develop/CHANGES.rst)
* [ ] Pull request includes screenshots or videos of UI changes
* [ ] Pull request includes a [sign off](https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.rst#sign-off)

### CODE REVIEW

File: ClearKeep/Modules/Search/CKSearchContactViewController.swift [hunk 1]
```diff
@@ -258,8 +258,8 @@ extension CKSearchContactViewController {
     private func cellForMatrix(_ indexPath: IndexPath) -> CKContactListMatrixCell {
         let cell = self.tableView.dequeueReusableCell(withIdentifier: CKContactListMatrixCell.identifier, for: indexPath) as! CKContactListMatrixCell
         
-        if let ds = self.filteredMatrixSource {
-            cell.displayNameLabel.text = ds[indexPath.row].displayName
+        if let ds = self.filteredMatrixSource, !ds.isEmpty {
+                    cell.displayNameLabel.text = ds[indexPath.row].displayName
```
Comment (HoaPham2711) [inline ClearKeep/Modules/Search/CKSearchContactViewController.swift]: Pls re-format this line

**Review by HoaPham2711 — CHANGES_REQUESTED** [review]
Minor change request

File: ClearKeep/Modules/Search/CKSearchContactViewController.swift (hunk 1 above)
Comment (TiemLuong) [reply ClearKeep/Modules/Search/CKSearchContactViewController.swift]: okay

**Review by HoaPham2711 — APPROVED** [review]
LGTM

**Pull request merged**